docs/conf/inspircd.conf.example

```diff
@@ -613,8 +613,38 @@
          # banned from the server.
          moronbanner="You're banned! Email abuse@example.com with the ERROR line below for help."
 
-         # exemptchanops: exemptions for channel access restrictions based on prefix.
-         exemptchanops="nonick:v flood:o"
+         # exemptchanops: Allows users with with a status mode to be exempt
+         # from various channel restrictions. Possible restrictions are:
+         #  - auditorium-see  Permission required to see the full user list of
+         #                    a +u channel (requires the auditorium module).
+         #  - auditorium-vis  Permission required to be visible in a +u channel
+         #                    (requires the auditorium module).
+         #  - blockcaps       Channel mode +B - blocks messages with too many capital
+         #                    letters (requires the blockcaps module).
+         #  - blockcolor      Channel mode +c - blocks messages with mIRC color codes
+         #                    (requires the blockcolor module).
+         #  - censor          Channel mode +G - censors messages based on the network
+         #                    configuration (requires the censor module).
+         #  - filter          Channel mode +g - blocks messages containing the given
+         #                    glob mask (requires the chanfilter module).
+         #  - flood           Channel mode +f - kicks (and bans) on text flood of a
+         #                    specified rate (requires the messageflood module).
+         #  - nickflood       Channel mode +F - blocks nick changes after a specified
+         #                    rate (requires the nickflood module).
+         #  - noctcp          Channel mode +C - blocks any CTCPs to the channel
+         #                    (requires the noctcp module).
+         #  - nonick          Channel mode +N - prevents users on the channel from
+         #                    changing nicks (requires the nonicks module).
+         #  - nonotice        Channel mode +T - blocks /NOTICEs to the channel
+         #                    (requires the nonotice module).
+         #  - regmoderated    Channel mode +M - blocks unregistered users from
+         #                    speaking (requires the services account module).
+         #  - stripcolor      Channel mode +S - strips mIRC color codes from
+         #                    messages (requires the stripcolor module).
+         #  - topiclock       Channel mode +t - limits changing the topic to (half)ops
+         # You can also configure this on a per-channel basis with a channel mode.
+         # See m_exemptchanops in modules.conf.example for more details.
+         exemptchanops="censor:o filter:o nickflood:o nonick:v regmoderated:o"
 
```

Comment:
Kept the addition of the restriction (mode) descriptions here to keep it easy when configuring.